config/renovate/renovate.json

```diff
@@ -180,5 +180,6 @@
     "branchPrefix": "konflux/mintmaker/"
   },
   "forkProcessing": "enabled",
+  "allowedPostUpgradeCommands": ["rpm-lockfile-prototype rpms.in.yaml"],
```

Comment:
I just wanted to ask if there is a difference between this and `"^rpm-lockfile-prototype rpms.in.yaml$"` from security point of view? The Renovate docs aren't very clear about this unfortunately.

Reply:
Updated with the suggestion.